src/types/misc.ts

```diff
@@ -131,9 +131,17 @@ export interface CellPosition {
   sheetId: UID;
 }
 
-// A border description is a pair [style, ]
-export type BorderStyle = "thin" | "medium" | "thick" | "dashed" | "dotted" | "double";
-export type BorderDescr = [BorderStyle, Color];
+export const borderStyles = ["thin", "medium", "thick", "dashed", "dotted", "double"] as const;
+export type BorderStyle = typeof borderStyles[number];
+// A complete border description is a pair [style, color]
+export type BorderDescr = { style: BorderStyle; color: Color };
+
+// A complete border(s) data is a set of position-color-style informations
+export type borderData = {
```

Comment:
Use a real docstring. Otherwise IDEs don't recognize the comment as a docstring (same above)
```suggestion
/**
 * A complete border(s) data is a set of position-color-style informations
 */
export type borderData = {
```

Reply:
up ^^